stackslib/src/clarity_vm/clarity.rs

```diff
@@ -1276,6 +1276,10 @@ impl<'a, 'b> ClarityBlockConnection<'a, 'b> {
         })
     }
 
+    pub fn initialize_epoch_3_0(&mut self) -> Result<Vec<StacksTransactionReceipt>, Error> {
+        todo!()
```

Comment:
This should just be `Ok(vec![])` for now: once there's actual changes necessary for epoch initialization, they can be added.

Reply:
Okay, updated!